web/index.html

```diff
@@ -24,8 +34,8 @@
 		<p style="display:inline" id="basic-stats"></p>
 
 		<canvas id="chart"></canvas>
-		<a href="https://github.com/klausenbusk/flathub-stats">https://github.com/klausenbusk/flathub-stats</a>
+		<a href="https://github.com/klausenbusk/flathub-stats" target="_blank" rel="noopener noreferrer">https://github.com/klausenbusk/flathub-stats</a>
```

Comment:
`noopener` is the default with `target="_blank"` so can be removed.

Reply:
Which one should I remove?

Reply:
I'm not sure exactly if it is true or not. I checked online and it's common to use all three together.

Reply:
The [standard](https://html.spec.whatwg.org/#following-hyperlinks) says `noopener` is implied when using `target="_blank"` and both Chrome and Firefox have shipped it: https://www.chromestatus.com/feature/6140064063029248 and https://bugzilla.mozilla.org/show_bug.cgi?id=1522083

So just remove `noopener` and it should be fine.